README.md

```diff
@@ -1,23 +1,31 @@
 ## Note on current project status
 
-Hi there!
+TLDR: MDTraj is currently undergoing a transition to new maintainers (the Folding@home consortium).
+Please bear with us as we slowly respond to issues and clean up.
 
-MDTraj is a project that I (@rmcgibbo) primarily developed between 2013 and 2016 while I was a
-graduate student. That was a long time ago! I've moved on, and no longer use MDTraj in my
-current role. Most of the other contributors from that time have similarly moved on. I do
-respond to github issues and pull requests from time to time, but my bandwidth is limited so
-don't expect to necessarily get a response on github issues from me. (Also **please** don't
-email me with support requests).
+Hello!
 
-MDTraj is looking for new maintainers! I would be happy to transfer ownership and responsibility
-of the code base to another group of maintainers. Please reach out to me if you're interested in
-taking over the project.
+For anybody wondering, the MDTraj repo is currently undergoing a transition to new maintainers;
+The Folding@home consortium (@FoldingAtHome) will now be taking over support, maintainence
+and management of MDTraj with @xuhuihuang leading the charge, and @apayne97 and myself (@sukritsingh)
```

Comment:
Feel free to mention me somewhere in here, then, thanks

Reply:
I've added you into the latest commit!